sql/src/test/java/io/crate/analyze/relations/SubselectRewriterTest.java

```diff
@@ -59,13 +72,42 @@ public void setupAnalyzer() throws Exception {
                 T3.T2_INFO.ident(), T3.T2_INFO
             )
         );
+
+        UserManager userManager = new UserManager() {
```

Comment:
Why don't we use the UnsupportedUserManager implementation if we want to work in a non-enterprise environment?
This "hybrid" implementation is odd - createUser should return a failed Future if there is no enterprise license, but here it returns null (same with others)